### PR TITLE
add runbook for ExtremelyHighIndividualControlPlaneCPU

### PR DESCRIPTION
**Pull request by tkashem**

### CODE REVIEW

File: alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# ExtremelyHighIndividualControlPlaneCPU
+
+## Meaning
+
+[These alerts][ExtremelyHighIndividualControlPlaneCPU] are triggered when
```
Comment (dgrisonnet) [inline alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md:5]:
```suggestion
[This alert][ExtremelyHighIndividualControlPlaneCPU] is triggered when
```

File: alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+  - find out the root cause of the degradation, and
+  - then determine the next steps accordingly.
+
+If this needs to be reported, then capture the following dataset, and file
+a new issue in BugZilla with links to the captured dataset
+
+- must-gather
+- audit logs
+- dump of prometheus data
```
[the remark this answers is not in the document]
Comment (tkashem) [reply alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md:69]: prom dump is needed (especially in this case) when reporter wants to create a BZ. Unfortunately, we have not yet productized  prom dump yet. this script is all we have for now to capture prom data.

File: alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+The following prometheus queries can be used to diagnose:
+```sh
+// CPU utilization of container(s) running on a particular node
+topk(5,
+  sum by (namespace, pod, container) (
+    irate (container_cpu_usage_seconds_total{node="NODE_NAME",container!="",pod!=""}[5m])
+  )
+)
+
+// CPU utilization on master nodes
```
Comment (dgrisonnet) [inline alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md:59]:
```suggestion
// CPU utilization of containers on master nodes
```

File: alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+sum by (node) (
+  irate (container_cpu_usage_seconds_total{container!="",pod!=""}[5m])
+    and on (node) cluster:master_nodes
+)
+
+// inspect cgroup
```
Comment (dgrisonnet) [inline alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md:65]:
```suggestion
// CPU utilization of the master nodes from the cgroups
```

File: alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+you will experience a cascade as the remaining kube-apiservers
+are also under-provisioned.
+
+To fix this, increase the CPU and memory on your control plane nodes.
+
+## Diagnosis
+
+The following prometheus queries can be used to diagnose:
+```sh
+// CPU utilization of container(s) running on a particular node
```
Comment (dgrisonnet) [inline alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md:52]:
```suggestion
// Top 5 of containers with the most CPU utilization on a particular node
```

File: alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
+# ExtremelyHighIndividualControlPlaneCPU
+
+## Meaning
+
+[This alert][ExtremelyHighIndividualControlPlaneCPU](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/assets/alerts/cpu-utilization.yaml)
```
Comment (dgrisonnet) [inline alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md:5]: Preview is broken here, you should change that to:
```suggestion
[This alert][ExtremelyHighIndividualControlPlaneCPU]
```
and add the following at the end of the file:
```
[ExtremelyHighIndividualControlPlaneCPU]: https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/assets/alerts/cpu-utilization.yaml
```

Same applies for `HighOverallControlPlaneCPU`.

File: alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
+# ExtremelyHighIndividualControlPlaneCPU
+
+## Meaning
+
+[This alert][ExtremelyHighIndividualControlPlaneCPU]is triggered when there
```
Comment (dgrisonnet) [inline alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md:5]:
```suggestion
[This alert][ExtremelyHighIndividualControlPlaneCPU] is triggered when there
```

**Pull request merged**